### DISCUSSION TITLE
After payload version upgrade from 1.15.0 to 2.13.0 can't fetch existing data

### DISCUSSION THREAD

**Discussion by komal1220** [payloadcms/payload#6170] [Q&A]
I have update the payload version from 1.15.0 to 2.13.0. After that when  run my project 
1. I can't see existing data of one of my collection in Admin UI.
2.I can see old collection data and version collection data in database. 
3. i have already versions:{drafts: true} property added in my collection file. when i comment out that code or set   versions: {
    drafts: false,
  },
  then I can see all the previously added data and new data.
  can you help me to solve this issue?

**Comment by shadow5og** [comment]
I am on version 3 and experiencing the same thing.
